<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(170, 85, 255);
background-color: rgb(170, 0, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="switch_but">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>普通用户登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 0, 127);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 85, 127);</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="1">
            <widget class="QLabel" name="login_err_label">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;

color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="username_label">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);
</string>
             </property>
             <property name="text">
              <string>用户名:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="username_lineEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 0, 127);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);
</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="password_label">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>密码:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="password_lineEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 0, 127);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="logon_but">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="login_but">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 255);
font: 700 9pt &quot;FiraCode Nerd Font Mono&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
